experimentation/src/main/java/com/automattic/android/experimentation/ExPlat.kt

```diff
@@ -0,0 +1,89 @@
+package com.automattic.android.experimentation
+
+import com.automattic.android.experimentation.ExPlat.RefreshStrategy.ALWAYS
+import com.automattic.android.experimentation.ExPlat.RefreshStrategy.IF_STALE
+import com.automattic.android.experimentation.ExPlat.RefreshStrategy.NEVER
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import org.wordpress.android.fluxc.model.experiments.Assignments
+import org.wordpress.android.fluxc.model.experiments.Variation
+import org.wordpress.android.fluxc.model.experiments.Variation.Control
+import org.wordpress.android.fluxc.store.ExperimentStore
+import org.wordpress.android.fluxc.store.ExperimentStore.Platform
+import org.wordpress.android.fluxc.utils.AppLogWrapper
+import org.wordpress.android.util.AppLog.T
+
+class ExPlat(
+    private val experiments: Set<Experiment>,
+    private val experimentStore: ExperimentStore,
+    private val appLogWrapper: AppLogWrapper,
+    private val coroutineScope: CoroutineScope,
+    private val isDebug: Boolean
+) {
+    private val platform = Platform.WORDPRESS_ANDROID
```

Comment:
I believe this should be declared by client

Reply:
Great catch! Thank you